alf/utils/common.py

```diff
@@ -433,6 +433,31 @@ def _cast(obs):
     return tf.nest.map_structure(_cast, observation)
 
 
+def traverse_transform_observation(obs, levels, field, func):
+    """
+    Args:
+        obs (nested Tensor|nested ndarray): observations to be applied the transformation
+        levels (list[str]): levels
```

Comment:
explain what are the valid values of `levels` and its relationship with `field`.